sycl/source/detail/graph_impl.hpp

```diff
@@ -1501,6 +1515,12 @@ class exec_graph_impl {
   bool MIsUpdatable;
   /// If true, the graph profiling is enabled.
   bool MEnableProfiling;
+  /// If true, the graph fusion is enabled.
+  bool MEnableFusion;
+  /// If true, the graph fusion is required.
+  bool MRequireFusion;
```

Comment:
Do we need these members as well as `MFinalizePropList` or can we just do `MFinalizePropList.has_property<property::graph::enable_fusion>()` or `MFinalizePropList.has_property<property::graph::require_fusion>()`  ?